i18n/translations/fr/form-builder.json

```diff
@@ -621,6 +621,7 @@
   "selectAutocompleteHint": "Cette option est utile lorsque les personnes qui remplissent le formulaire fournissent des informations personnelles telles que leur adresse, courriel ou numéro de téléphone.",
   "selectElement": "Sélectionnez un type d'élément",
   "send": "Envoyer",
+  "copy": "[FR]copy",
```

Comment:
@anikbrazeau this string is used in the title text when an element is duplicated e.g. "My element copy"